floris/simulation/flow_field.py

```diff
@@ -101,6 +101,10 @@ def initialize_velocity_field(self, grid: Grid) -> None:
         self.v_sorted = self.v_initial_sorted.copy()
         self.w_sorted = self.w_initial_sorted.copy()
 
+        self.u = self.u_sorted.copy()
```

Comment:
@bayc Is this valid? I think `flow_field.u` and etc are meant to be the unsorted versions.

Reply:
Ah my bad! @bayc the `steady_state_atmospheric_solver` is not called when `no_wake=True`, and so the `finalize()` commands are never called. That then gives an error because `.u`, `.v` and `.w` are undefined.

Reply:
I see! I'll dig into this a bit and see if I can come up with a solution.

Reply:
Thank you @bayc. Another idea I had is to have `no_wake` rely on the `None` submodels. It'll be slower, but still very manageable I would think.